packages/eslint-config-mc-app/index.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
   plugins: [
     'import',
     'jest',
+    'jest-dom',
```

Comment:
I think adding only the plugin won't work because all the rules are disabled by default, as far as I recall. We can either enable them manually or use the [recommended configuration](https://github.com/testing-library/eslint-plugin-jest-dom#recommended-configuration).

Reply:
Thanks. I missed this.

Reply:
9535232